packages/runtime/container-runtime/src/connectionTelemetry.ts

```diff
@@ -144,7 +168,7 @@ class OpPerfTelemetry {
             assert(this.opSendTimeForLatencyStatistics !== undefined,
                 0x120 /* "Undefined latency statistics (op send time)" */);
 
-            const duration = Date.now() - this.opSendTimeForLatencyStatistics;
+            this.durationRoundtripTime = Date.now() - this.opSendTimeForLatencyStatistics;
```

Comment:
I'm not sure I see the need to have this.durationRoundtripTime as a member variable, not a local variable as it used to be

Reply:
My bad - it was a change I forgot to revert.